runtime/vm/jvminit.c

```diff
@@ -2392,7 +2393,11 @@ static UDATA checkArgsConsumed(J9PortLibrary* portLibrary, J9VMInitArgs* j9vm_ar
 		if (IS_CONSUMABLE( j9vm_args, i ) && !IS_CONSUMED( j9vm_args, i )) {
 			char* optString = j9vm_args->actualVMArgs->options[i].optionString;
 			char* envVar = j9vm_args->j9Options[i].fromEnvVar;
-
+			
+			/* If an option is empty or contains only whitespace, it should be ignored instead of rejected */
+			if (isEmpty(optString)) {
```

Comment:
can `optString` be null here?  I see an explicit null check on it on line 2402 before calling `strncmp` on it...

Reply:
Using https://github.com/eclipse/openj9/files/3035727/sampleJNI.zip and adding a `NULL` string to the args array `options[0].optionString = NULL;` results in a seg fault for both the RI and the J9 implementations.

However, it's probably best to add the explicit null check anyways - will be adding this in.